test/powershell/engine/Module/ModulePath.Tests.ps1

```diff
@@ -88,21 +88,14 @@ Describe "SxS Module Path Basic Tests" -tags "CI" {
             $newModulePath = & $powershell -nopro -c '$env:PSModulePath'
             $paths = $newModulePath -split [System.IO.Path]::PathSeparator
 
-            if ($IsWindows)
-            {
-                $paths.Count | Should -Be 4
-            }
-            else
-            {
-                $paths.Count | Should -Be 3
-            }
+            $paths.Count | Should -Be 4
 
             $paths[0] | Should -Be $expectedUserPath
             $paths[1] | Should -Be $expectedSharedPath
             $paths[2] | Should -Be $expectedSystemPath
```

Comment:
If we expect 4 paths, do we need a test for `$paths[3]`?

Reply:
yes, it'll be the inherited fake one